src/components/comment.vue

```diff
@@ -1,15 +1,15 @@
 <template>
-  <ion-row ref="doubleTapRef">
-    <ion-col size="1">
+  <ion-row ref="doubleTapRef" class="" v-if="!comment.replyTo">
```

Comment:
May I suggest flipping the order of these two rows as it allows for a nice `v-if=... v-else` that is easier to read. Once you've moved the other row above, this can change to:

```suggestion
  <ion-row ref="doubleTapRef" v-else>
```